src/components/utils/Error404.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import bgVideo from '../../assets/video/A32NX.mp4';
+
+export const Error404 = () => {
+    return (
+        <>
+            <div className="mx-auto my-80 max-w-md md:max-w-7xl text-center ">
+                <h5 className="text-9xl text-blue-light font-bold">404</h5>
+                <h2 className="mt-12 text-4xl font-bold">
```

Comment:
```suggestion
                <h2 className="mt-8 text-4xl font-bold">
```

Slightly reduce margin top between text and title